templates/registration/email_change_activate_email.html

```diff
@@ -0,0 +1,24 @@
+<div style="border:2px solid #fd0;margin:4px 0;"><div style="background:#000;border:6px solid #000;">
```

Comment:
Is this HTML copied from somewhere?

Reply:
Yes, it's copied from the password reset/account activation emails.